templates/repo/find/files.tmpl

```diff
@@ -0,0 +1,21 @@
+{{template "base/head" .}}
+<div class="page-content repository">
+	{{template "repo/header" .}}
+	<div class="ui container">
+		<div class="df ac">
+			<a href="{{$.RepoLink}}">{{.RepoName}}</a>
+			<span class="mx-3">/</span>
+			<div class="ui input f1">
+				<input id="repo-file-find-input" type="text" autofocus data-url-data-link="{{.DataLink}}" data-url-tree-link="{{.TreeLink}}">
+			</div>
+		</div>
+		<table id="repo-find-file-table" class="ui single line table">
+			<tbody>
+			</tbody>
+		</table>
+		<div id="repo-find-file-no-result" class="ui row center mt-5" style="display:none;">
```

Comment:
Isn't there already the `hidden` styleclass for that?
```suggestion
		<div id="repo-find-file-no-result" class="hidden ui row center mt-5">
```
Of course, that should also be changed in the JS then...

Reply:
I do not think there is a common `hidden` CSS class.  I do not think it will hide the elements.

Could you help to confirm if it works?

Reply:
Nope, in most cases, it doesn't work .........

And you must grep `class="[^"]*hidden[^"]*"`,  `.*` is incorrect ........

Reply:
Hmm..... I think we can use `<div id="repo-find-file-no-result" class="ui row center mt-5" hidden>`  instead, the `[hidden]` is defined in `semantic.css`, which should be safe.

Reply:
> `ag 'class=".*hidden' templates/ | wc -l` at least gives 68 results. Either most of them are incorrect, or there is a `hidden` class...

Good question, after reading the related code/document again, I opened a new issue:
* https://github.com/go-gitea/gitea/issues/19926